uberfire-widgets/uberfire-widgets-commons/pom.xml

```diff
@@ -99,6 +99,13 @@
       <scope>provided</scope>
     </dependency>
 
+    <!-- Test -->
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
```

Comment:
Please use `mockito-core`, because `mockito-all` is uberjar and will be in near future banned (you will get enforcer error during build)